operation-and-usage/operations/out-of-band-interoperation.md

```diff
@@ -187,6 +187,18 @@ This, however, brings up another question: what should happen if you redeploy th
 
 Specifying `manage-by-service` not only updates device configuration in the CDB with the out-of-band value, it also adds the value under service instance's out-of-band changes (also called extra operations). NSO takes these changes into account when calculating service configuration after mapping code runs. It allows the service to preserve an out-of-band value during a redeploy. Additionally, it ties the value to the lifecycle of the service; if the service is deleted, so is the out-of-band configuration.
 
+
+It might be desirable to abort the out of band handling entirely and fail the transaction with an out-of-sync error if certain out of band changes are detected on a device. This can be achieved using the `abort` action, for example:
```

Comment:
```suggestion
It may be desirable to abort out-of-band handling entirely and fail the transaction with an out-of-sync error if certain out-of-band changes are detected on a device. This can be achieved using the `abort` action, for example:
```